.github/workflows/main.yml

```diff
@@ -860,7 +860,7 @@ jobs:
         # workaround for https://bugs.launchpad.net/ubuntu/+source/llvm-defaults/+bug/1972855
         sudo mkdir -p /usr/lib/local/lib/python3.10/dist-packages/lldb
         sudo ln -s /usr/lib/llvm-15/lib/python3.10/dist-packages/lldb/* /usr/lib/python3/dist-packages/lldb/
-        cargo test --test all -- --ignored --test-threads 1 debug::
+        cargo test --release --test all -- --ignored --test-threads 1 debug::
```

Comment:
If these tests are fast enough mind adding both `--release`-and-not?

Reply:
Added. Numbers follow.

The tests themselves are fairly quick (because there are so few of them :)), what takes its time is building everything. 

This `--release` version now takes ~8min, and the previous version took ~5min. Combining these, we conservatively get ~13min, which is on the order of the longest job in this PR (~12min).

Reply:
Ah ok good point, in that case let's go ahead and drop the debug build then and only test `--release`. Mind adding a comment as to why we're only testing `--release` as well?